src/components/FormItems/TextField.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import MUITextField from 'material-ui/TextField';
+import { FormControl, FormHelperText } from 'material-ui/Form';
+
+const TextField = ({ error, ...props }) => (
+  <FormControl className="SignUp__field" error={!!error}>
```

Comment:
`classname` `SignUp__field` suggests a `SignUp` component, yet the filename/path suggest generic TextField. If you intend to keep the `FormItems` generic, you should change the classname and have the CSS files inside each `FormItems` component.